packages/matter.js/src/environment/MdnsService.ts

```diff
@@ -32,15 +32,16 @@ export class MdnsService {
             const vars = environment.get(VariableService);
             const network = environment.get(Network);
 
-            const enableIpv4 = options?.ipv4 ?? environment.get(VariableService).boolean("mdns.ipv4") ?? true;
+            const netInterface = vars.get("mdns.networkInterface", options?.networkInterface);
+            const enableIpv4 = vars.boolean("mdns.ipv4") ?? options?.ipv4 ?? true;
             this.#broadcaster = await MdnsBroadcaster.create(network, {
                 enableIpv4,
-                multicastInterface: vars.get("mdns.discoverInterface", options?.discoverInterface),
```

Comment:
I thought it was nice allowing for announcement and discovery on different interfaces but nbd

Reply:
We decided that several time back on master that it makes no sense to have it different. ;-)